server/src/test-fast/java/com/thoughtworks/go/server/service/EntityHashingServiceTest.java

```diff
@@ -73,6 +74,16 @@ public void shouldRegisterToListenForConfigChange() {
         verify(goConfigService).register(entityHashingService);
     }
 
+    @Test
+    public void shouldUseObjectHashCodeForPluginSettings() {
+        PluginSettings pluginSettings = new PluginSettings();
+        String expectedMd5 = CachedDigestUtils.md5Hex(String.valueOf(pluginSettings.hashCode()));
```

Comment:
@prateekbaheti Would it be possible to write a test with a known hash, so that if something gets added, the test would fail and the person making the change can decide whether it should affect the hash?

Reply:
Makes sense, I will change that.